lib/ansible/modules/cloud/ovirt/ovirt_hosts.py

```diff
@@ -386,6 +392,13 @@ def main():
                 post_action=lambda h: time.sleep(module.params['poll_interval']),
                 fail_condition=failed_state,
             )
+        elif state == 'iscsidiscover':
+            ret = hosts_module.action(
+                # This is a synchronize action and we only want to
+                # know if we got 200 response code
+                action='iscsi_discover',
+                iscsi=otypes.IscsiDetails(address=module.params['iscsi']['address'])
```

Comment:
> Should this be enough? 
nope, I should also add it username and password
>  Also what will happen if that action was already invoked, do we get error, or HTTP 200?
If the discover was already invoke we will get HTTP 200, same as will be done through the REST.

> The best behavior would be to not sent iscsi_dicover if it was already sent. And sent it only when needed, when it wasn't yet sent.
But what if the user added targets to its storage server, and he would still want to call discover to discover the new targets.